resources/web/style/guide_base_styles.pcss

```diff
@@ -0,0 +1,25 @@
+#guide {
```

Comment:
maybe just `base`? everything in the docs is "guide".

Reply:
renamed!